pkg/services/baremetal/baremetal/baremetal.go

```diff
@@ -61,6 +61,8 @@ const (
 	// requeueAfter gives the duration of time until the next reconciliation should be performed.
 	requeueAfter = time.Second * 30
 
+	requeueAfterNoAvailableHost = time.Minute * 3
```

Comment:
IMO this could be increased further, especially since we will be able to catch the events from the hosts

Reply:
set to 7 minutes.